src/popover/docs/readme.md

```diff
@@ -10,7 +10,7 @@ There are three versions of the popover: `uib-popover` and `uib-popover-template
   Takes text only and will escape any HTML provided for the popover body.
   
 * `uib-popover-html`
-  Takes an expression that evaluates to an html string. *The user is responsible for ensuring the content is safe to put into the DOM!*
+  Takes an expression that evaluates to an html string.  Note that this HTML is not compiled.  If compilation is required, please use the `uib-popover-template` attribute option instead. *The user is responsible for ensuring the content is safe to put into the DOM!*
```

Comment:
Super minor, but should only use 1 space after a period - browsers eat up any extra spacing.